src/utils.ts

```diff
@@ -105,3 +105,7 @@ export function getDBNetworks(network: Network): DBNetwork[] {
 
   return []
 }
+
+export function formatMana(mana: string, maximumFractionDigits = 2): string {
+  return (Number(mana || '0') / 1e18).toFixed(maximumFractionDigits).toLocaleString()
```

Comment:
If I'm not wrong, mana here is expressed in wei, right? If that's the case, the number can grow pretty big. We might want to load it using either ethers or BigInt, to later convert it into ether. Number won't be able to hold such number.